scss/_navbar.scss

```diff
@@ -69,8 +69,10 @@
 
 .navbar-brand {
   float: left;
-  padding-top:    .25rem;
-  padding-bottom: .25rem;
+  padding-top: (($navbar-item-padding-y + 0.425rem) / $font-size-lg) + rem;
+  padding-bottom: (($navbar-item-padding-y + 0.425rem) / $font-size-lg) + rem;
+  margin-top: $navbar-item-margin-y;
+  margin-bottom: $navbar-item-margin-y;
```

Comment:
Properties should be ordered float, padding-top, padding-bottom, margin-top, margin-right, margin-bottom, font-size